libs/examples/src/chart/chart-behavior-switch-example/chart-behavior-switch-example.ts

```diff
@@ -15,112 +15,110 @@
  */
 
 import { Component } from '@angular/core';
-
 import { generateData } from '../chart-data-utils';
 
+const cpuUsageSeries: Highcharts.IndividualSeriesOptions[] = [
+  {
+    name: 'CPU usage',
+    type: 'line',
+    data: generateData(40, 20, 35, 1370304000000, 900000),
+    color: '#92d9f8',
+  },
+  {
+    name: 'Number of process group instances',
+    type: 'column',
+    yAxis: 1,
+    data: generateData(40, 1, 4, 1370304000000, 900000),
+    color: '#006bba',
+  },
+];
+
+const retransmissionSeries: Highcharts.IndividualSeriesOptions[] = [
+  {
+    name: 'Number of retransmissions',
+    type: 'column',
+    data: generateData(40, 20, 35, 1370304000000, 900000),
+    color: '#fff5b7',
+  },
+  {
+    name: 'Server usage',
+    type: 'line',
+    yAxis: 1,
+    data: generateData(40, 15, 100, 1370304000000, 900000),
+    color: '#f5d30f',
+  },
+];
+
+const connectivitySeries: Highcharts.IndividualSeriesOptions[] = [
```

Comment:
I think we should move these array back into the class as a prop so it appears in the example source in barista.